tasks/setup_RedHat.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: Install SELinux Python dependencies.
   package:
-    name: libsemanage-python
+    name: "{{ 'libsemanage-python' if ansible_distribution_major_version|int < 8 else 'libsemanage-python3' }}"
```

Comment:
Please use the "version" comparator instead, see https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

Reply:
✔️